src/components/UI/PhotoFeature.js

```diff
@@ -61,16 +65,12 @@ const PhotoFeature = props => {
         <div css={!isHover ? frontShow : frontHide}>
           <img
             alt="image description"
-            src={
-              imageUrl
-                ? imageUrl
-                : "https://bulma.io/images/placeholders/480x480.png"
-            }
+            src={imageUrl}
+            onError={loadPlaceholderImage}
```

Comment:
So this actually fires eh?  Maybe it does...I've just never seen it.  Can you add some supporting documentation on this?  If it's a common usage I'm fine with it, but if it's a non-standard workaround I'd prefer again to handle this a different way.

Reply:
Yes, it fires this. We can test by passing invalid image source/by adding a string to image source on `PhotoBox` and `PhotoFeature` components and it will fire. Here's a screenshot from when I tested this
<img width="1231" alt="Screen Shot 2020-07-14 at 12 41 46 PM" src="https://user-images.githubusercontent.com/14085957/87552285-19886b00-c677-11ea-86c9-70fd2fffb801.png">